GNUmakefile

```diff
@@ -152,9 +155,9 @@ srpm-%:	export $(PACKAGE)-%.spec
 	case "$(WITH)" in 	\
 	  *pre_release*)	\
 	    sed -i 's/global\ pcmk_release.*/global\ pcmk_release\ 0.%{specversion}.%{shortcommit}.git/' $(PACKAGE).spec;	\
-	    sed -i 's/Version:.*/Version:\ $(shell echo $(NEXT_RELEASE) | sed -e s:Pacemaker-:: -e s:-.*::)/' $(PACKAGE).spec;;	\
+	    sed -i 's/pcmkversion:.*/pcmkversion:\ $(shell echo $(NEXT_RELEASE) | sed -e s:Pacemaker-:: -e s:-.*::)/' $(PACKAGE).spec;;	\
 	  *)			\
-	    sed -i 's/Version:.*/Version:\ $(shell git describe --tags $(TAG) | sed -e s:Pacemaker-:: -e s:-.*::)/' $(PACKAGE).spec;;\
+	    sed -i 's/pcmkversion:.*/pcmkversion:\ $(shell git describe --tags $(TAG) | sed -e s:Pacemaker-:: -e s:-.*::)/' $(PACKAGE).spec;;\
```

Comment:
the spec file doesn't have a colon after pcmkversion -- i think you want to match the full '^%global pcmkversion' here